README.md

```diff
@@ -50,54 +59,78 @@ However you can pass in some config options, the defaults are:
   -- Normal mode mapping left hand side
   line_mapping = "gcc",
   -- Visual/Operator mapping left hand side
-  operator_mapping = "gc"
+  operator_mapping = "gc",
+  -- Hook function to call before commenting takes place
+  hook = nil
 }
 ```
 
-**Ignore Empty Lines**
+#### Ignore Empty Lines
```

Comment:
again happy with these as just bold

Reply:
Was reported by linter as something along the line: emphasis used in place of header. I agree with linter as emphasis such as bold text probably fit more within the sentence to emphasise it's part.